src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -526,6 +527,7 @@ void uct_rc_mlx5_iface_fill_attr(uct_rc_mlx5_iface_common_t *iface,
                                srq->verbs.srq);
         break;
     case UCT_IB_MLX5_OBJ_TYPE_DEVX:
+    case UCT_IB_MLX5_OBJ_TYPE_NULL:
```

Comment:
why UCT_IB_MLX5_OBJ_TYPE_NULL is not an empty operation?

Reply:
good question, I don't know why SRQ obj type defines QP attrs for TX too :)